trellis/core/subscriber.hpp

```diff
@@ -132,15 +132,14 @@ class SubscriberImpl {
 
   void SetCallbackWithWatchdog(Callback callback, WatchdogCallback watchdog_callback, unsigned watchdog_timeout_ms,
                                EventLoop event_loop) {
-    Timer watchdog_timer{nullptr};
-    auto callback_wrapper = [this, callback, watchdog_callback, watchdog_timer, watchdog_timeout_ms, event_loop](
+    auto callback_wrapper = [this, callback, watchdog_callback, watchdog_timeout_ms, event_loop](
                                 const char* topic_name_, const trellis::core::TimestampedMessage& msg_, long long time_,
                                 long long clock_, long long id_) mutable {
-      if (watchdog_timer == nullptr) {
+      if (watchdog_timer_ == nullptr) {
```

Comment:
@jdgoldberg FYI I had to make this change because after bumping eCAL, every invocation of this callback would have a `nullptr` valued `watchdog_timer`. I suspect it has to do with an internal change in eCAL in how they retain a copy of this lambda object.

Reply:
Cool makes sense. Nice catch!